sdk/src/main/java/com/relaypro/sdk/Relay.java

```diff
@@ -241,10 +275,10 @@ private MessageWrapper sendRequest(Map<String, Object> message, boolean waitForP
      * @param target the device that you would like to start an interaction with.
      * @param name a name for your interaction
      * @return any errors received from the server.
-     * @see #startInteraction(String, String, Object)
+//     * @see #startInteraction(String, String, Object)
      */
     @SuppressWarnings("UnusedReturnValue")
-    public String startInteraction(String target, String name) {
+    public String startInteraction(Object target, String name) {
```

Comment:
It always feels a bit odd to have the type as Object, but I don't know if there is a better way to do it. Maybe some kind of class that is a wrapper for all the possibilities?